client/src/components/Sponsors/READ ME.txt

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
rename the file to README.md instead of READ ME.md

Reply:
Changed this too.